xml/banner.xml

```diff
@@ -2,9 +2,8 @@
 
 <banner>
 
-Have you heard?
-<link url="https://nginx.org/en/">nginx</link> has moved to
-<link url="https://github.com/nginx/nginx/">GitHub</link>.
-<link url="https://mailman.nginx.org/pipermail/nginx/2024-September/R7KBUUNTG3B2QAAMPS6M7YDOSVCMOZD7.html">Read more</link>.
+Celebrating 20 years of nginx — from the
+<link url="https://hg.nginx.org/nginx/rev/551102312e19">first public version</link>
```

Comment:
There will be a celebratory blog post on blog.nginx.org - suggest linking to that instead.

Reply:
referencing mercurial commit looks awkward after moving to git

Reply:
20 years ago it was mercurial so I think it's ok here.

Reply:
ok, link changed to git: https://github.com/nginx/nginx/commit/0e8348c500bac5303a3b7f2ffee4b7ded05558ff